pygfx/renderers/wgpu/_renderer.py

```diff
@@ -641,15 +648,15 @@ def get_pick_info(self, pos):
 
         # Collect data from the buffer
         data = self._pixel_info_buffer.map_read()
-        color = tuple(data[0:4].cast("B"))
+        rgba = tuple(data[0:4].cast("B"))
```

Comment:
why not convert to Color on this line?

Reply:
Yes, and let's fix that `Color` expects 0..1 at the same time 😆